### ISSUE TITLE
Wrong offence detected?

### ISSUE THREAD

**Issue by jatap** [rubocop/rubocop#214]
New 0.8.0 version throws this offence:

C: 76: Space inside hash literal braces missing.

``` ruby
# spec/routing/routing_spec.rb
it { expect(get: "/#{locale.to_s}/#{fake_path}").not_to be_routable }
```

_0.7.2 marks as valid_
